vector/src/main/java/im/vector/util/RoomUtils.java

```diff
@@ -655,8 +658,14 @@ public boolean onMenuItemClick(final MenuItem item) {
                                 break;
 
                             case R.id.ic_action_notifications_mute:
-                                moreActionListener.onUpdateRoomNotificationsState(session,
-                                        room.getRoomId(), BingRulesManager.RoomNotificationState.MUTE);
+                                BingRulesManager.RoomNotificationState state;
+                                if (pushManager.useFcm() && room.isEncrypted() && !room.isDirect()) {
```

Comment:
Please add a comment here to explain: 
"The mention only option is not supported for the encrypted room on Tchap-Android (GooglePlay variant). That's why we could not suggest it to the user, but this option seems to be the most appropriate when they decide to mute the notifications for a room. By selecting "Mention_Only" here, the notifications will be disabled on Mobile clients, but the user will keep receiving notifications on Tchap-Web in case of mention."

Reply:
Added